src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/ocrvalidation/client/OcrValidationClient.java

```diff
@@ -4,23 +4,32 @@
 import org.springframework.http.HttpHeaders;
 import org.springframework.stereotype.Component;
 import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
 import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.req.FormData;
 import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.res.ValidationResponse;
 
 @Component
 public class OcrValidationClient {
 
+    public static final String PATH = "/forms/{form-type}/validate-ocr";
+
     private final RestTemplate restTemplate;
 
     public OcrValidationClient(RestTemplate restTemplate) {
         this.restTemplate = restTemplate;
     }
 
-    public ValidationResponse validate(String url, FormData formData, String s2sToken) {
-
+    public ValidationResponse validate(String baseUrl, FormData formData, String s2sToken) {
```

Comment:
Doesn't it require a change in the call, too?

Reply:
currently no URLs are configured